presto-main/src/main/java/io/prestosql/operator/annotations/FunctionsParserHelper.java

```diff
@@ -119,6 +115,9 @@ public static boolean containsImplementationDependencyAnnotation(Annotation[] an
                         verifyTypeSignatureDoesNotContainAnyTypeParameters(typeSignature, typeSignature, typeParameterNames);
                     }
                 }
+                else {
+                    throw new IllegalArgumentException("Unsupported operator type " + operator);
```

Comment:
Is this what a user/admin will see in the logs when starting up the server? If so, we should have a clearer message.